### ISSUE TITLE
Firefox crashing

### ISSUE THREAD

**Issue by schauppi** [slidevjs/slidev#1827]
I updated slidev to version 0.49.26 and suddenly it stopped working in Firefox (I don't know the previous version number). In Safari it works like a charm. I open Firefox, go to http://localhost:3030/ and it gets stuck on the loading screen and Firefox crashes. 

1. terminal command: slidev
2. Open Firefox on http://localhost:3030/ (also with /presenter and /overview)

**Environment**

- Slidev version: 0.49.26 (global)
- Browser: Firefox 129.0.1
- OS: MacOS Sonoma 14.5

**Comment by KermanX** [comment]
#1826?

**Comment by schauppi** [comment]
> #1826?

Firefox is not based on chromium?

**Comment by KermanX** [comment]
So it seems that this bug happens in MacOS + Chromium/Firefox🧐

**Comment by twitwi** [comment]
I tested on linux (firefox and chromium), with a fresh pnpm create, I got the issue.
I could see the title slide but with a memory usage of around 1GB then I tried the slide overview and it froze my graphical session.
This looked liked a memory issue but I checked and had a lot of remaining memory, maybe it is a kind of video memory issue.

I thought some slides of the demo were memory hungry, so I trimmed down the slide.md to a single slide, same issue.

I will be trying to manually bisect to get the first version for which it occurs (I'll be back after a few crashes :D).
